src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/ComponentTestBase.kt

```diff
@@ -30,7 +30,7 @@ import java.util.Base64
 @SpringBootTest(webEnvironment = RANDOM_PORT)
 @ActiveProfiles("db-test")
 @TestInstance(PER_CLASS)
-abstract class ComponentTestBase {
+abstract class ComponentTestBase(private val startGotenbergMockServer: Boolean = true) {
```

Comment:
Proabably unimportant but ...
`private val startGotenbergMockServer: Boolean = true` 
feels to me a bit more like
`private val useRealGotenbergServer: Boolean = false`
?

Reply:
Yep sounds sensible